ddtrace/tracer/option.go

```diff
@@ -111,7 +116,7 @@ func defaults(c *config) {
 	if v := os.Getenv("DD_SERVICE"); v != "" {
 		c.serviceName = v
 	} else {
-		c.serviceName = filepath.Base(os.Args[0])
+		c.serviceName = DefaultServiceName
```

Comment:
We should think of a good way to make sure that the fact that the service name exists in two places doesn't become confusing. One idea is to clearly document that the service in the `globalconfig` package refers strictly to integrations.

Reply:
This is also no longer an issue with the new implementation.